src/user.spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jest */
+
+import {
+    getUser,  
+    testConnection,
+    xo,
+  } from "./util";
+  
+  describe("user", () => {
+    let userId1, userId2;
+  
+    test("getAll() works", async () => {
+      expect.assertions(1);
+      const users = await xo.call("user.getAll");
+      expect(users).toBeInstanceOf(Array);
+    });
+  
+    describe(".create()", () => {
+      test("create user <email, password>", async () => {
+        expect.assertions(2);
+  
+        userId1 = await xo.call("user.create", {
+          email: "wayne1@vates.fr",
+          password: "batman1",
+        });
+        expect(typeof userId1).toBe("string");
+  
+        const user = await getUser(xo, userId1);
+        expect(user).toEqual(
+          {
+            id: userId1,
+            email: "wayne1@vates.fr",
+            groups: [],
+            preferences: {}
+          }
+        );
+  
+        await testConnection({
```

Comment:
IMHO, the connection tests should not be in this file.

Reply:
My bad, in this file but in a dedicated block.

Reply:
Ok :+1: